cockroach/sql/structured.proto

```diff
@@ -0,0 +1,110 @@
+// Copyright 2015 The Cockroach Authors.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+//  implied. See the License for the specific language governing
+//  permissions and limitations under the License. See the AUTHORS file
+//  for names of contributors.
+//  Author: Tamir Duberstein (tamird@gmail.com)
+
+syntax = "proto2";
+
+import "cockroach/sql/privilege.proto";
+package cockroach.sql;
+
+option go_package = "sql";
+
+message ColumnType {
+  // These mirror the types supported by the sql/parser. See
+  //  sql/parser/types.go.
+  enum Kind {
+    BOOL = 0;
+    INT = 1;
+    // INT(width)
```

Comment:
Hmm, these comments are on the previous line [in the original](https://github.com/cockroachdb/cockroach/blob/master/sql/structured.proto#L35). Is these easy to fix?